### PR TITLE
Add GTM Monitor SOAP support

### PR DESCRIPTION
**Pull request by gabrielfortin**

### CODE REVIEW

File: library/f5bigip_gtm_monitor_soap.py [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+        description:
+            - Specifies the type of monitor you want to use to create the new monitor.
+        default: soap
+    description:
+        description:
+            - User defined description.
+    destination:
+        description:
+            - Specifies the IP address and service port of the resource that is the destination of this monitor.
+        default: *:*
```
Comment (erjac77) [inline library/f5bigip_gtm_monitor_soap.py]: You need to put the destination value between single quotes ('*:*').

File: library/f5bigip_gtm_monitor_soap.py [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+        description:
+            - Specifies the number of seconds the target has in which to respond to the monitor request.
+        default: 120
+    url_path:
+        description:
+            - Specifies the URL for the Web service that you are monitoring, for example, /services/myservice.aspx.
+    username:
+        description:
+            - Specifies the username, if the monitored target requires authentication.
+notes:
```
Comment (erjac77) [inline library/f5bigip_gtm_monitor_soap.py]: You can remove the notes.

File: library/f5bigip_gtm_monitor_soap.py [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+    - Requires BIG-IP software version >= 11.6
+requirements:
```
Comment (erjac77) [inline library/f5bigip_gtm_monitor_soap.py]: requirements:
    - BIG-IP >= 12.0
    - ansible-common-f5
    - f5-sdk

**Comment by gabrielfortin** [comment]
Fixed, thank you.

**Pull request merged**